source/state/swap/api.ts

```diff
@@ -0,0 +1,181 @@
+/////////////////////////
+// Imports
+/////////////////////////
+
+import { createAsyncThunk } from "@reduxjs/toolkit";
+
+/////////////////////////
+// Types
+/////////////////////////
+
+import {
+  IStageTransaction,
+  ISearchResponse,
+  ISearchCurrency,
+  ICurrencyRate,
+  IPendingTransaction,
+  IExolixTransaction
+} from "./types";
+import { RootState } from 'state/store';
+
+
+/////////////////////////
+// Constants
+/////////////////////////
+
+const API_KEY = 'Zm5w8D384z4E9Yma24Adn24obD';
+const SWAP_BASE_URL = 'https://api.lattice.exchange/swapping';
```

Comment:
This should go in .env also. I would assume we'd want to use staging for non-prod environments.